contributors/devel/welcome-to-kubernetes-new-developer-guide.md

```diff
@@ -0,0 +1,128 @@
+
+Welcome to Kubernetes! (New Developer Guide)
+============================================
+
+2017-06-21 through 2017-08-06
+Original author: [Alexander Campbell](https://github.com/alexandercampbell)
+
+---
+
+_This document assumes that you know what Kubernetes does. If you don't, search
+YouTube for ["Kubernetes"](https://www.youtube.com/watch?v=8SvQqZNP6uo) and try
```

Comment:
I think you should give this link as one of the examples instead of hyperlink. When we say search for "Kubernetes" in youtube - the correct link will be something like https://www.youtube.com/results?search_query=kubernetes